scripts/tools/SetupLtiConsumer.php

```diff
@@ -0,0 +1,146 @@
+     */
+    protected function provideOptions()
+    {
+        return [
+            'key' => array(
+                'prefix' => 'k',
+                'longPrefix' => 'key',
+                'description' => 'Consumer key',
+                'required' => true,
+                'cast' => 'string'
+            ),
+            'secret' => array(
+                'prefix' => 's',
+                'longPrefix' => 'secret',
+                'description' => 'Consumer secret',
+                'required' => true,
+                'cast' => 'string'
+            ),
+            'label' => array(
+                'prefix' => 'l',
+                'longPrefix' => 'label',
+                'description' => 'Consumer label',
+                'required' => false,
+                'cast' => 'string'
+            ),
+            'callbackUrl' => [
+                'prefix' => 'cu',
+                'longPrefix' => 'callbackUrl',
+                'description' => 'Callback url for consumer',
+                'required' => false
+            ]
+        ];
+    }
+
+    /**
+     * Provides description of the script
+     *
+     * @return string
+     */
+    protected function provideDescription()
+    {
+        return 'Script to create a sample consumer';
+    }
+
+    /**
+     * Provides help of this script
+     *
+     * @return array
+     */
+    protected function provideUsage()
+    {
+        return [
+            'prefix' => 'h',
+            'longPrefix' => 'help',
+            'description' => 'Prints the help.'
+        ];
+    }
+
+    /**
+     * Run the script
+     */
+    protected function run()
+    {
+        if (empty($this->getOption('key')) || empty($this->getOption('secret'))) {
+            return new Report(Report::TYPE_ERROR, 'Not all arguments were provided. Try to run the script with -h option');
+        }
+
+        $consumerService = $this->getClassService();
+        $clazz = $this->getClass(ConsumerService::CLASS_URI);
+
+        $label = $this->hasOption('label') ? $this->getOption('label') : $consumerService->createUniqueLabel($clazz);
+
+        try {
+            $consumer = $consumerService->createInstance($clazz, $label);
+        } catch (\Exception $e) {
+            return new Report(Report::TYPE_ERROR, 'Error while creating consumer. Actual message is: ' . $e->getMessage());
+        }
+
+        $consumer->setPropertyValue($this->getProperty(DataStore::PROPERTY_OAUTH_KEY), $this->getOption('key'));
+        $consumer->setPropertyValue($this->getProperty(DataStore::PROPERTY_OAUTH_SECRET), $this->getOption('secret'));
+
+        if ($this->hasOption('callbackUrl')) {
+            $consumer->setPropertyValue($this->getProperty(DataStore::PROPERTY_OAUTH_CALLBACK), $this->getOption('callbackUrl'));
+        }
+
+        $consumerLabel = $consumer->getLabel();
+
+        return new Report(Report::TYPE_SUCCESS, 'Lti consumer \'' . $consumerLabel .'\' was created. Check GUI to edit its properties');
```

Comment:
```suggestion
        return new Report(Report::TYPE_SUCCESS, sprintf('Lti consumer "%s" was created. Check GUI to edit its properties', $consumer->getLabel()));
```
And remove useless `$consumerLabel` variable.